docs/ai/quickstarts/snippets/text-to-image/hosting/Program.cs

```diff
@@ -21,18 +21,20 @@
 });
 // </SnippetAddImageGenerator>
 
+// <SnippetConfigureOptions>
+imageBuilder.ConfigureOptions(options =>
+{
+    options.MediaType = "image/png";
+}).UseLogging();
+// </SnippetConfigureOptions>
+
 WebApplication app = builder.Build();
 
 // <SnippetUseImageGenerator>
 // Use the image generator in an endpoint.
 app.MapPost("/generate-image", async (IImageGenerator generator, string prompt) =>
 {
-    var options = new ImageGenerationOptions
-    {
-        MediaType = "image/png"
-    };
-
-    ImageGenerationResponse response = await generator.GenerateImagesAsync(prompt, options);
+    ImageGenerationResponse response = await generator.GenerateImagesAsync(prompt);
     DataContent dataContent = response.Contents.OfType<DataContent>().First();
 
     return Results.File(dataContent.Data.ToArray(), "image/png");
```

Comment:
I missed on my first review, but this might slightly better:
```C#
    return Results.File(dataContent.Data.ToArray(), dataContent.MediaType);
```

Reply:
I'll submit a PR.